docs/camino-node/auto-upgrade.md

```diff
@@ -0,0 +1,129 @@
+Checking latest release... latest camino-node version is: v1.0.0
+Getting current running node version... current running camino-node version is: v0.4.9
+Current Node version (v0.4.9) is NOT the latest (v1.0.0)
+Upgrading camino-node...
+Stopping service...done @Thu Jul 27 22:36:02 UTC 2023
+Node version found.
+Attempting to download: https://github.com/chain4travel/camino-node/releases/download/v1.0.0/camino-node-linux-amd64-v1.0.0.tar.gz
+
+     0K .......... .......... .......... .......... ..........  0% 1.51M 19s
+    50K .......... .......... .......... .......... ..........  0% 2.03M 17s
+   100K .......... .......... .......... .......... ..........  0% 4.15M 13s
+   150K .......... .......... .......... .......... ..........  0% 8.02M 11s
+--------------------8<--------------------
+ 29300K .......... .......... .......... .......... .......... 99% 11.1M 0s
+ 29350K .......... .......... .......... .......... .......... 99% 8.85M 0s
+ 29400K .......... .......... .......... .......... .......... 99% 8.59M 0s
+ 29450K .......... .......... .......... .......... .......... 99% 13.7M 0s
+ 29500K ..........                                            100% 18.5M=3.4s2023-07-27 22:36:07 URL:https://objects.githubusercontent.com/github-production-release-asset-2e65be/492036145/6f1d02ce-373c-4369-b277-
+ada7df6ce5af?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20230727%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20230727T223602Z&X-Amz-Expires=300&X-Amz-Signature=0b206b166738adff6cde104
+079b7926e8e43b1e01abbd4c4f09882a8f006207d&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=492036145&response-content-disposition=attachment%3B%20filename%3Dcamino-node-linux-amd64-v1.0.0.tar.gz&response-cont
+ent-type=application%2Foctet-stream [30218484/30218484] -> "camino-node-linux-amd64-v1.0.0.tar.gz" [1]
+Unpacking node files...
+camino-node-v1.0.0/tools/
+camino-node-v1.0.0/tools/cert
+camino-node-v1.0.0/LICENSE
+camino-node-v1.0.0/plugins/
+camino-node-v1.0.0/camino-node
+Node files unpacked into /home/camino/camino-node
+Node upgraded, starting service...done @Thu Jul 27 22:36:08 UTC 2023
+New node version:
+camino-node: v1.0.0, commit: c403cb6
+caminogo: v1.0.0-rc1, commit: 0c5035f69
+  compat: v1.0.0 [database: v1.4.5]
+Done!
+```
+
+If the latest released node version is the same as the one already installed, the output will look like this:
+
+```
+*** Starting upgrade process for user: camino ***
+*** Date is: Thu Jul 27 22:37:01 UTC 2023 ***
+Found camino-node systemd service...
+Preparing environment...
+Found amd64 architecture...
+Checking latest release... latest camino-node version is: v1.0.0
+Getting current running node version... current running camino-node version is: v1.0.0
+Current node version (v1.0.0) is the latest (v1.0.0). Skipping...
+```
+
+## Docker Compose
+
+If you have installed camino-node with Docker Compose use the steps below to configure auto upgrade.
+
+**_ !!! TODO !!! _**
```

Comment:
We can't keep this in our docs. Maybe remove the whole section docker compose and then add it later.

Reply:
I will not merge until Docker section is finished.